app/templates/main/userProfile.html

```diff
@@ -237,102 +238,215 @@ <h5>{{volunteer.firstName}} {{volunteer.lastName}} is the manager of:</h5>
+      <div class="accordion-body">
+        <div class="table-responsive">
+          <table class="table table-striped" id="bgHistoryTable">
+            <thead>
+              <tr>
+                <th scope="col">Description</th>
+                 {% if g.current_user.isCeltsAdmin%}
+                   <th scope="col">Status</th>
+                   <th scope="col">Date</th>
+                   <th class="" id="displaySave" scope="col"></th>
+                 {% endif %}
+               </tr>
+             </thead>
+             <tbody>
+               {% for bgType in backgroundTypes %}
+               <tr>
+                 <td>
+                     <h6>{{bgType.description}}</h6>
+                     <ul class="mb-0" id="bgHistory{{bgType.id}}" >
+                        {% for bgStatus in allBackgroundHistory[bgType.id] %}
+                         <li>
+                           {{bgStatus.backgroundCheckStatus}}: {{bgStatus.dateCompleted.strftime("%m/%d/%Y")}}
+                           <button class="bg-transparent border-0 text-danger" id="deleteBgHistory" data-id="{{bgStatus.id}}">X</button>
+                         </li>
+                        {% endfor %}
+                     </ul>
+                 </td>
+                 {% if g.current_user.isCeltsAdmin %}
+                    <td>
+                        <select class="passedBackgroundCheck form-select" id="{{bgType.id}}" data-id="{{bgType.id}}">
+                            <option value=''></option>
+                            <option value="Submitted">Submitted</option>
+                            <option value="Passed">Passed</option>
+                            <option value="Failed">Failed</option>
+                        </select>
+                    </td>
+                    <td><input name="dateCompleted" type="date" id="{{bgType.id}}_date" class="form-control"></td>
+                    <td><button type="button" class="savebtn btn btn-primary" id="{{bgType.id}}" data-id="{{bgType.id}}" data-username="{{volunteer.username}}">Save</button></td>
+                {% endif %}
+                </tr>
+              {%endfor%}
+             </tbody>
+          </table>
         </div>
       </div>
     </div>
-    {% endif %}
-    <!-- End Background Check Table -->
-</div>
-<!-- Ban or Unban Modal -->
-<div class="modal fade" id="banModal" tabindex="-1" aria-labelledby="banVolunteerModalLabel" aria-hidden="true" style="width:-60px;">
-  <form onsubmit="return validate();">
-    <div class="modal-dialog">
-      <div class="modal-content">
-        <div class="modal-header">
-          <h5 class="modal-title">Ban Volunteer</h5>
-          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+  </div>
+  {% endif %}
+  <!-- End Background Check Table -->
+  <!-- Start Notes section -->
+  <div class="accordion-item" id="accordion5">
+    <h3 class="accordion-header" id="headingSix">
```

Comment:
Why are you giving it the div id accordion5 and the accordion header id headingSix? It is the sixth accordion on the page.